docs/editor/profiles.md

```diff
@@ -128,6 +128,211 @@ Profiles can be used to customize VS Code for students to ease the use in a clas
+* [Ruff](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff) - Integrates the [Ruff](https://github.com/charliermarsh/ruff) Python linter.
+
+This profile also sets the following settings:
+
+```json
+    "python.analysis.autoImportCompletions": true,
+    "python.analysis.fixAll": ["source.unusedImports"],
+    "workbench.colorTheme": "Default Dark+ Experimental",
+    "editor.defaultFormatter": "ms-python.black-formatter"
+```
+
+### Data Science Profile Template
+
+The Data Science profile is a good starting point for all data and notebook work. It comes with specific snippets and has the following extensions:
+
+* [Data Wrangler](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.datawrangler) - Data cleaning and preparation for tabular datasets.
+* [Black Formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter) - Formatting support using the [black](https://github.com/python/black) formatter.
+* [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) - Jupyter notebook support, interactive programming and computing that supports Intellisense, debugging and more.
+* [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) - IntelliSense, linting, formatting, debugging, refactoring.
+* [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) - Open any folder or repository inside a Docker container and take advantage of Visual Studio Code's full feature set.
+* [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) - Your AI pair programmer.
+
+This profile also sets the following settings:
+
+```json
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter",
+        "editor.formatOnType": true,
+        "editor.formatOnSave": true
+    },
+    "editor.inlineSuggest.enabled": true,
+    "editor.lineHeight": 17,
+    "breadcrumbs.enabled": false,
+    "files.autoSave": "afterDelay",
+    "jupyter.themeMatplotlibPlots": true,
+    "jupyter.widgetScriptSources": [
+        "unpkg.com",
+        "jsdelivr.com"
+    ],
+    "notebook.experimental.outputScrolling": true,
+    // "notebook.outline.showCodeCells": true,
+    "workbench.colorTheme": "Default Dark+ Experimental",
+    "files.exclude": {
+        "**/.csv": true,
+        "**/.parquet": true,
+        "**/.pkl": true,
+        "**/.xls": true
+    }
+```
+
+### Doc Writer Profile Template
+
+The Doc Writer profile is a good lightweight setup for writing documentation. It comes with the following extensions:
+* [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) - Spelling checker for source code.
+* [GitHub Markdown Preview](https://marketplace.visualstudio.com/items?itemName=bierner.github-markdown-preview) - Changes VS Code's built-in markdown preview to match GitHub.
```

Comment:
bierner.github-markdown-preview is an extension pack that includes most of the extension below.
I don't think there is a need to include it in the list.
A general question is when a profile template is using an extension pack, should the extension pack be shown in the Profile view?

Reply:
Good question. I think it is fine showing it, since we never know if the extnesion pack contributes code itself. Right @sandy081 

As for the docs list. I think we can either have the pack, or list each extension. But not both. Whatever you think is better @gregvanl

Reply:
Yeah, showing it is fine. It is also useful to uninstall the complete pack at once.